### ISSUE TITLE
Dropdown - Add support for accessibility

### ISSUE THREAD

**Issue by PasiVaisanen** [sebgroup/green#1218]
### Description

Angular dropdown component don't have support for aria tags e.g. aria-describedby, aria-labelledby, aria-invalid

### Who will benefit from this feature?

All screen reader users.

**Comment by splashdust** [comment]
@PasiVaisanen This is actually by design! :)

You may have heard the saying "No ARIA is better than bad ARIA". This is a simplified way of saying that incorrect use of ARIA can often decrease accessibility rather than increase it. Use of `aria-*`-attributes can be notoriously difficult to get right. The specs are fuzzy and behaviour differs slightly between different versions and vendors of assistive technologies.

Therefore, the approach we take with Green is to build these things into the component internals instead, to the extent it is possible.
For example, the Dropdown component uses Combobox and Listbox roles and related aria-attributes internally. It also sets up a `<label>` with `for`-attribute, applies `aria-invalid` when relevant, etc.
This way, you don't have to think about this when consuming the component. It should be wired up mostly correct out of the box. And if we do find problems or mistakes, we can fix it in the component, and everyone using it gets the same fix for free.

Another consideration is that we are migrating to web components with Shadow DOM. This means that you can't associate internal parts of a component to IDs on the outside through aria-attributes, which further necessitates this internal setup.

**TLDR;** aria-attributes are built-in, not necessary to add manually.

I'm closing this issue, but if you have a specific issue where you found a component is lacking in a11y, feel free to open more issues and we can deal with it specifically!
